SwiftWisdomTests/StandardLibrary/String/String+IndexingTests.swift

```diff
@@ -2,6 +2,30 @@ import XCTest
 
 class String_IndexingTests: XCTestCase {
     
+    func testRangeSubscriptReturnsProperString() {
+        XCTAssertEqual("hello"[0..<0], "")
```

Comment:
If we're doing something funky w.r.t. string length and indexing, sure.  Otherwise, aren't we just adding a test that Swift's string indexing works with multibyte characters?

Also I assume that *any* high-byte Unicode character would do for testing?